### PR TITLE
Added Aesry Hunting Support

### PR DESCRIPTION
**Pull request by elBlanquito**
Fixed Map DB where broken (where I identified deficiencies) and added moss meys, bloodwolves&bobcats, frostweaver&snowbeasts, snow goblins1, 2, &3, yvhh la'tami, snaer hafwa, ghoul crows, wir dinego & greater ghoul raven, and graverobbers!

### CODE REVIEW

**Comment by elBlanquito** [comment]
I have throne city on paper, will push it later on after i go back and double check my mapdb work in faldesu inlet!

**Comment by Sarvatt** [comment]
thanks for marking the rooms and doing this

File: data/base-hunting.yaml [hunk 1]
```diff
@@ -737,17 +890,6 @@ hunting_zones:
   - 10810
   - 10811
   - 10812
-  # https://elanthipedia.play.net/Armored_shalswar 300-400
```
Comment (rpherbig) [inline data/base-hunting.yaml]: Why were these removed?

File: data/base-mining.yaml [hunk 2]
```diff
@@ -99,3 +99,58 @@ mining_buddy_rooms:
   - 9711
   - 9712
   - 9713
+  
+  # https://elanthipedia.play.net/RanikMap98a
```
Comment (rpherbig) [inline data/base-mining.yaml]: This was added in #1817, you may need to merge master back into this branch.

Comment (elBlanquito) [inline data/base-mining.yaml]: I messed up as well, i'll cancel this and re-do!

File: data/base-hunting.yaml (hunk 1 above)
Comment (elBlanquito) [inline data/base-hunting.yaml]: It was an accident I was in the middle of trying to fix it. I'm learning bare with me, sorry!

**Comment by rpherbig** [comment]
No problem :)

**Comment by elBlanquito** [comment]
kk, got the merge master!

**Comment by rpherbig** [comment]
The `shalswar` hunting zone occurs twice. The mining file looks like you're adding new content.

**Comment by elBlanquito** [comment]
Having trouble understanding what guidance you are offering based on your second sentence? Would you please clariify?

**Comment by rpherbig** [comment]
I wouldn't expect to see any changes to `mining-base.yaml` in this PR, since those changes were already merged in #1817. I'm afraid merging this might cause a duplicate to appear in the file.

**Comment by elBlanquito** [comment]
I see. Yes that is problematic. My current intent is to understand the workflow better here, need to learn so much more! hahah

**Comment by elBlanquito** [comment]
How could i go about sending a pull request from my fork that doesn't also include that?

**Comment by elBlanquito** [comment]
hahah clearly not. well... we ruled that out!

**Comment by elBlanquito** [comment]
re-opened as #1819
